app/src/app/classes/stix/relationship.ts

```diff
@@ -170,7 +226,7 @@ export class Relationship extends StixObject {
             // this.source_name = raw.source_object.stix.name;
             
             let src_sdo = raw.source_object.stix;
-            if ("external_references" in src_sdo) {
+            if ("external_references" in src_sdo && src_sdo["type"] !== "x-mitre-data-component") {
```

Comment:
Shouldn't the `src_sdo.external_references[0].hasOwnProperty("external_id")` call in the next `if` statement catch this already?

Also worth noting #245 adds relevant functionality (that PR may be merged before this one, in which case adjustments may need to be made): https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/pull/245/files#diff-b90cce5e706eb76c909962cd43080303de93e3b4c215c4b046d31eedbf570ec4R40